src/hooks/useServerData.ts

```diff
@@ -57,7 +57,7 @@ const useServerData = (
     shouldFetch
       ? [
           `/discord/server/${serverId}/${option?.memberCountDetails}`,
-          { method: "POST" },
+          { method: "GET" },
```

Comment:
I think you can completely remove this option object here, since the default method is `GET` anyways. Then, you can also remove the square brackets around the fetch URL and just pass it as a single template string, so the code will be much cleaner.

Reply:
Ah, you probably right, but I just already merged it into main. We will fix it later, I guess.